catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/TemplateTransformer.java

```diff
@@ -97,11 +128,30 @@ public Metacard toQueryTemplateMetacard(Map<String, Object> formTemplate) {
       if (querySettings != null) {
         metacard.setQuerySettings(querySettings);
       }
+
+      Result result = metacardAlreadyExists(id, filterBuilder, catalogFramework);
+
+      // Perform update of old Metacard if already exists
+      if (result != null) {
+        Metacard oldMetacard = null;
+        oldMetacard = result.getMetacard();
+        oldMetacard.setAttribute(new AttributeImpl(Core.MODIFIED, new Date()));
```

Comment:
❗️ Looks like we might regress by not adding the old created date to this metacard
https://github.com/codice/ddf/pull/3186/files#diff-0cff128da7b84e6e61b5370f3c0a5ec5L301